File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.py
def solution(my_string):
    answer = ''
    for i in range(len(my_string)):
        if my_string[i:i+1].islower() :
            answer += my_string[i:i+1].upper()
        else :
            answer += my_string[i:i+1].lower()
    return answer

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    var answer = '';
    for(let char of my_string){
        const char_check = char.toLowerCase();
        if (char == char_check){
            const charUpper = char.toUpperCase();
            answer += charUpper;
        }else{
            answer += char_check;
        }
    }
    return answer;
}